### ISSUE TITLE
Clarify which color emojis are supported

### ISSUE THREAD

**Issue by RazrFalcon** [pop-os/cosmic-text#3]
TrueType provides at least:

- `sbix`
- `CBDT` + `CBLC`
- `svg`
- `COLR` v0 and v1

**Comment by jackpot51** [comment]
Swash supports everything but COLR v1 and the heic encoding for sbix

**Comment by RazrFalcon** [comment]
I'm pretty sure `swash` doesn't support SVG and `heic`. And only `png` in `sbix`.

**Comment by jackpot51** [comment]
Is there an SVG font sample I should try?

**Comment by RazrFalcon** [comment]
Sure: https://github.com/adobe-fonts/emojione-color

**Comment by dfrg** [comment]
Yes, swash doesn’t support svg. It could expose the table but parsing and composition would be out of scope for the library. My understanding is that COLRv1 basically supercedes this anyway
